app/src/js/components/Collections/granules.js

```diff
@@ -104,7 +105,7 @@ const CollectionGranules = ({
     return bulkActions(granules, actionConfig);
   }
 
-  function selectWorkflow(selector, selectedWorkflow) {
```

Comment:
Why are we getting rid of `selector` here?

Reply:
@dopeters The linter suggested to remove because it was imported but never declared so it's not being used. It was old code that was there before. I can put it back.

Reply:
@dopeters This is what I get: <img width="595" alt="Screen Shot 2021-07-27 at 11 10 36 AM" src="https://user-images.githubusercontent.com/40603352/127179515-57750446-6e68-42ab-a959-83e1aaac06b3.png">

Reply:
I think we need that for the function params to be correct. To get rid of the lint error, we can name it `_selector` instead.